card.js

```diff
@@ -0,0 +1,27 @@
+export function createCharacterCard(datatvshow) {
+  const newCard = document.createElement("li");
+  ul.append(newCard);
```

Comment:
this will not work as ul is not defined.
And the job of the createCharacterCard function is it not to append the card to the container. Just to return the card as element. So returning the newCard constant is also missing.

Reply:
Ok, I changed it. thanks